### PR TITLE
Microverse Mobile Portfolio Project

### PR DESCRIPTION
**Pull request by Nicolaswg**
This is the first pull request from the Microverse mobile portfolio.

In this Project, you can see different types of tools used for a professional portfolio mobile template.

**New Tools:**

1. Figma templates
2. Mobile layout

**Project Starting Point:**

- Basic HTML and CSS files
- Linter folder
- Figma Template Resources
- HTML and CSS code Skeleton

**Project Description goals**

1. Created the first 2 sections of the Microverse Figma Template.
2. Learn how to navigate the templates of Figma.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        <!-- Header toolbar -->
+        <nav>
+            <div class="content header-toolbar-container">
+                <div class="logo-container">
+                    <div class="logo-item">
+                        <h5 class="logo">My Logo</h5>
+                    </div>
+                </div>
+                <div class="menu-container">
+                    <div class="menu-item">
```
Comment (Alexoid1) [inline index.html:22]: - [ ] Replace div with a ul tag and for each item use li tag that has a link tag that will redirect to the corresponding section of the page.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+</head>
+<body>
+    <!-- Project Header -->
+    <header>
+        <!-- Header toolbar -->
+        <nav>
+            <div class="content header-toolbar-container">
+                <div class="logo-container">
+                    <div class="logo-item">
+                        <h5 class="logo">My Logo</h5>
```
Comment (Alexoid1) [inline index.html:18]: - [ ] Replace h5 with a link tag. The logo the most of the time is a link that redirects to the homepage in this case the intro section.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                <h6 class="link-item"><a href="#" class="link">LET'S CONNECT</a></h6>
+            </div>
+            <div class="network-container">
+                <div class="network-items">
+                    <div class="icon twitter"></div>
+                    <div class="icon linkedlin"></div>
+                    <div class="icon medium"></div>
+                    <div class="icon github"></div>
+                    <div class="icon peace"></div>
+                </div>
```
Comment (Alexoid1) [inline index.html:53]: - [ ] Replace div with a ul tag and for each social media item use li tag that has a link tag that will redirect to your different social media profiles.

**Review by Alexoid1 — CHANGES_REQUESTED** [review]
# Hi @Nicolaswg,

Impressive project!
You are doing a great job so far, and the project is looking good.
- No linter errors.
- Descriptive README.MD file.
- Follow Git-hub flow.

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes :recycle:
- For the project to get approved, also implement the changes mentioned below.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (Nicolaswg) [reply index.html:22]: In this menu, I only have empty divs that represent the hamburger icon made by myself :)

Comment (sheylaPozo) [reply index.html]: Hello @Nicolaswg  Good job at implementing the changes requested by previous reviewer 👍🌝

**Comment by sheylaPozo** [comment]
## Optional Suggestions 
---

- [x] Would look better if you fix the linter errors following these steps :
- - name: Setup Webhint
        run: |
          npm install --save-dev hint@6.x
          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.hintrc
      - name: Webhint Report
        run: npx hint --telemetry=off     

        Happy coding!!  👍 💯 

tenor

**Comment by sheylaPozo** [comment]
## Optional Suggestions  ♻☺
---

- [ ] As an optional suggestion, it would look better if you fix the linter errors following these steps :
-  Try Running the next npm on your console :  
 ```
   npm install --save-dev hint@6.x[ -f .hintrc ]
   npx hint --telemetry=off 
```

```
      - name: Setup Webhint
        run: |
          npm install --save-dev hint@6.x
          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.hintrc
      - name: Webhint Report
        run: npx hint --telemetry=off     
```

  >  Happy coding!!  👍 💯 

tenor

**Pull request merged**